charts/stable/vaultwarden/ci/ct-values.yaml

```diff
@@ -0,0 +1,3 @@
+persistence:
+  config:
+    enabled: true
```

Comment:
shouldn't this be an empty-dir?

Reply:
Why should this be empty dir? That would guarantee config loss. there is more than just templated config here. its also the data dir for Vaultwarden

Reply:
This is the testing values.yaml ;-)
It always looses its config :P

Reply:
Thank you, been staring at a lot of text today